teamscale-gradle-plugin/src/main/kotlin/com/teamscale/config/extension/TeamscaleTestImpactedTaskExtension.kt

```diff
@@ -0,0 +1,37 @@
+package com.teamscale.config.extension
+
+import com.teamscale.TestImpacted
+import com.teamscale.config.AgentConfiguration
+import com.teamscale.config.TestwiseCoverageConfiguration
+import org.gradle.api.Action
+import org.gradle.api.Project
+import org.gradle.testing.jacoco.plugins.JacocoTaskExtension
+import java.io.Serializable
+
+/**
+ * Holds all user configuration regarding testwise coverage report uploads.
+ */
+open class TeamscaleTestImpactedTaskExtension(
+    val project: Project,
+    jacocoExtension: JacocoTaskExtension,
+    testImpacted: TestImpacted
```

Comment:
IMHO the term "TestImpacted" does not make it clear what this class/abstraction actually is. Are these tests? Are these files?

Reply:
`TestImpacted` is a task class. Renamed the parameter, but renaming the class would be a breaking change as that might be used in end-user-scripts.